sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroSource.java

```diff
@@ -229,7 +229,7 @@
 
   private AvroSource(String fileNameOrPattern, long minBundleSize, String schema, Class<T> type,
       String codec, byte[] syncMarker) {
-    super(fileNameOrPattern, minBundleSize);
+    super(fileNameOrPattern, /* TODO */ true, minBundleSize);
```

Comment:
I've added the option as a builder API where the pattern already exists. Most sources are fully-configured within their constructors, so I've added it there. 

I generally worry about bloating the parameter list for constructors, but I also don't like maintaining many constructor overloads to handle default parameters. We could convert long parameter lists to an `Options` object. I didn't do that yet because I don't think any of these tip the scale. Let me know what you think.